lucille-core/src/main/java/com/kmwllc/lucille/core/Runner.java

```diff
@@ -164,23 +164,26 @@ public static Map<String, TestMessenger> runInTestMode(String configName) throws
 
   private static Map<String, List<Exception>> logValidation(Config config) throws Exception {
     Map<String, List<Exception>> exceptions = validatePipelines(config);
-    if (exceptions.isEmpty()) {
```

Comment:
exceptions is going to be a map from connector name to a list of exceptions for that connector
we want to print "Configuration is valid" if all connectors have an empty list of exceptions
so, instead of doing exceptions.isEmpty() we need a condition that checks isEmpty() for every value in the map